docs/en/observability/monitor-nginx.asciidoc

```diff
@@ -0,0 +1,249 @@
+[[monitor-nginx]]
+= Monitor Nginx: Observe the logs and metrics of your Nginx instances
+++++
+<titleabbrev>Monitor Nginx</titleabbrev>
+++++
+
+****
+**New to Elastic?** Follow the steps in our {estc-welcome}/getting-started-observability.html[getting started guide] instead of the steps described here.
+Return to this tutorial after you've learned the basics.
+****
+
+With the https://docs.elastic.co/integrations/nginx[Nginx Elastic integration] and the {agent}, you can collect valuable metrics and logs from your Nginx instances. Once this data is in {kib}, built-in dashboards and tools like Logs Explorer allow you to visualize and monitor your Nginx data from one place. This data provides valuable insight into your Nginx instances. For example, you might find:
```

Comment:
I think this can be tighter. I'm against "you can" but it might just be a me thing.

```suggestion
Use the https://docs.elastic.co/integrations/nginx[NGINX Elastic integration] and {agent} to collect valuable metrics and logs from your NGINX instances. Then, use built-in dashboards and tools like Logs Explorer in {kib} to visualize and monitor your NGINX data from one place. This data provides valuable insight into your Nginx instances—for example:
```